client/server/middleware/unsupported-browser.js

```diff
@@ -2,6 +2,23 @@ import config from '@automattic/calypso-config';
 import { addQueryArgs } from 'calypso/lib/url';
 import analytics from 'calypso/server/lib/analytics';
 
+/**
+ * The locales that are supported in exempting browser support checks.
+ */
+const SUPPORTED_LOCALES = [ 'en', ...config( 'magnificent_non_en_locales' ) ];
+
+/**
+ * The paths that are allowed even if the browser is unsupported.
+ */
+const ALLOWED_PATHS = [ 'browsehappy', 'themes', 'theme', 'calypso' ];
```

Comment:
I was surprised `/plugins` is not part of this list, as it's one of the other paths we currently server-render.

Reply:
> I was surprised `/plugins` is not part of this list, as it's one of the other paths we currently server-render.

I'd be open to adding it if we think it makes sense. For now I'd just opted to maintain the existing behavior.

Reply:
Yep, sounds good to keep as-is for the time being.